app/views/home/index.html.erb

```diff
@@ -1,2 +1,14 @@
 <h1>Home Page</h1>
-<p>Home page for employee accounts</p>
\ No newline at end of file
+<div class="card text-center mt-5">
+  <div class="card-header">
+    Title: Kudo App
+  </div>
+  <div class="card-body">
+    <h4 class="card-title">
+      Content: An application through which you can express your praise to other users
+    </h4>
+  </div>
+  <div class="card-footer">
+    <%= Time.now.strftime("%d of %B, %Y") %>
+  </div>
+</div>
```

Comment:
staraj się dodawać pustą linijkę na końcu plików, można ustawić sobie IDE aby robiło to za ciebie (szukaj pod add new line)